packages/react/src/VideoPlayer/components/controls/Controls.tsx

```diff
@@ -0,0 +1,304 @@
+const getMinuteSecondTime = (time: number) => {
+  const minutes = Math.floor(time / 60)
+  const seconds = Math.floor(time - minutes * 60)
+  const x = minutes < 10 ? `0${minutes}` : minutes
+  const y = seconds < 10 ? `0${seconds}` : seconds
+
+  return `${x}:${y}`
+}
+
+export const Controls = ({
+  videoRef,
+  videoWrapperRef,
+  totalTime,
+  currentTime,
+  playing: playingProp,
+  trackInformation,
+}: ControlsProps) => {
+  const [playing, setPlaying] = useState(playingProp)
+  const [playedTime, setPlayedTime] = useState(0)
+  const [volume, setVolume] = useState(0.5)
+  const [previousVolume, setPreviousVolume] = useState(volume)
+  const [closedCaption, setClosedCaption] = useState(true)
+  const [fullScreen, setFullScreen] = useState(false)
+
+  /* 1. Play / Pause                                            */
+  useEffect(() => {
+    if (videoRef.current) {
+      if (playing) {
+        videoRef.current.play()
+      } else {
+        videoRef.current.pause()
+      }
+    }
+  }, [videoRef, playing])
+
+  useEffect(() => {
+    setPlaying(playingProp)
+  }, [playingProp])
+
+  const renderPlayPauseButton = useMemo(() => {
+    return (
+      <IconControl
+        onClick={() => {
+          setPlaying(!playing)
+        }}
+        className={styles.VideoPlayer__shiftTooltipRight}
+        tooltip={!playing ? 'Play Video' : 'Pause Video'}
+      >
+        {!playing ? (
+          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
+            <path
+              d="M1.56489 23.8112C0.884817 24.2389 1.07491e-06 23.7501 1.0398e-06 22.9467L8.28679e-08 1.05473C4.73246e-08 0.241596 0.904067 -0.245404 1.58307 0.201969L18.5829 11.4026C19.2032 11.8113 19.1935 12.7244 18.5647 13.1198L1.56489 23.8112Z"
+              fill="currentColor"
+            />
+          </svg>
+        ) : (
+          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
+            <rect width="7.2" height="24" rx="1" fill="currentColor" />
+            <rect x="12.5" width="7.2" height="24" rx="1" fill="currentColor" />
+          </svg>
+        )}
+      </IconControl>
+    )
+  }, [playing])
+
+  /* 2. Seek                                                    */
+
+  useEffect(() => {
+    setPlayedTime(videoRef.current?.currentTime || 0)
+  }, [videoRef, videoRef.current?.currentTime])
+
+  const renderSeek = useMemo(() => {
+    return (
+      <>
+        <Range
+          type="range"
+          min="0"
+          max={totalTime || 0}
+          step={0.0001}
+          onInput={e => {
+            if (videoRef.current) videoRef.current.currentTime = e.currentTarget.valueAsNumber
+          }}
+          value={currentTime}
+          className={styles.VideoPlayer__progressBar2}
+          tooltip
+          tooltipFormatter={value => getMinuteSecondTime(value as number)}
+          id="VideoPlayer__Seek"
```

Comment:
With multiple videos on the same page, this would cause duplicate id violations, can we make this unique or remove it? I can't see where it's being referenced, so not sure if being used.

Reply:
This has been updated in this SHA: https://github.com/primer/brand/pull/373/commits/3476819281a2ff27b50715bf57261befc6f1d0f5